src/Swashbuckle.AspNetCore.Swagger/SwaggerMiddleware.cs

```diff
@@ -15,19 +18,29 @@ public class SwaggerMiddleware
         private readonly RequestDelegate _next;
         private readonly SwaggerOptions _options;
         private readonly TemplateMatcher _requestMatcher;
+#if !NETSTANDARD
+        private readonly TemplateBinder _templateBinder;
+#endif
 
         public SwaggerMiddleware(
             RequestDelegate next,
-            SwaggerOptions options)
+            SwaggerOptions options
+#if !NETSTANDARD
+            ,TemplateBinderFactory templateBinderFactory
```

Comment:
I just noticed this is a binary-breaking change. Unless we park this until v5, this should instead add a new constructor overload if there's a way to _somehow_ get this to work without the `TemplateBinderFactory`.

Reply:
I added a new constructor in 2fd5bcf0233579574954324aec268c20131fd0cb and restored the existing one in order to avoid a binary breaking change.

I think it should be considered to make the `SwaggerMiddleware` class internal for the next major version as I'm pretty sure everybody uses either the `UseSwagger` or `MapSwagger` extension method and nobody instantiates a `SwaggerMiddleware` instance directly.

Reply:
I've opened #2805 to track this suggestion.